doc/content/getting-started/guides/build-an-end-to-end-solution-gn/_index.md

```diff
@@ -0,0 +1,205 @@
+---
+title: "Building an End-to-End LoRaWAN Application with The Things Stack"
+description: ""
+weight: 
+---
+
+In this section, we will show how to build an end-to-end solution with [{{% tts %}}](/getting-started/what-is-tts/) to control the [Kuando Busylight](https://busylight.com/busylight-iot/) with the [Generic Node Sensor Edition](https://www.thethingsshop.com/products/generic-node-sensor-edition)(GNSE). 
+
+# Registering The Things Indoor Gateway
```

Comment:
```suggestion
## Registering The Things Indoor Gateway
```

Reply:
add one more hash, otherwise these don't show up on the right navigation bar. this goes for the whole doc